docs/src/layouts/links.js

```diff
@@ -356,7 +356,7 @@ const ComponentsLinks = [
         id: 'MAP',
         route: routes.MAP,
         children: 'Map',
-        tags: ['web', 'native'],
+        tags: ['web', 'native', 'ios', 'android'],
```

Comment:
Great spot on this for not having `iOS`

Reply:
We should get rid if these tags eventually and use content of each component docs as a single source of truth about the platforms the component exist on :)